notebooks/Add GPS Metadata Fields_FS.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #71.            if ([operation for operation in operations if operation == 'updateDefinition']):
</pre><p>I don't get what this is trying to do. shouldn't operations always be an empty list when this line is run?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>